src/main/java/uk/gov/hmcts/reform/sandl/snlrules/model/Session.java

```diff
@@ -38,6 +38,10 @@ public Session(String id, String judgeId, String roomId, OffsetDateTime start,
         this.sessionType = sessionType;
     }
 
+    public OffsetDateTime getStart() {
```

Comment:
can't we use Lombok here?

Reply:
yeah maybe we could, this is a code written before we started to use lombok.